docs/release.rst

```diff
@@ -1,23 +1,16 @@
 Release notes
 =============
 
-1.0.0-beta-10
-------------
+1.0.0 (2024-12)
+---------------
 
-Added
-^^^^^
+Features
+^^^^^^^^
 
-- Schema generation supports Hierarchical Associations 
-  referencing Provision Agreements
-
-
-1.0.0-beta-1
-------------
-
-Added
-^^^^^
-
-- Core SDMX model classes
-- Sync and async clients to retrieve metadata
-  from an SDMX Registry or SDMX-REST service
-- Helper functions to handle SDMX URNs
+- Offer core domain classes for the SDMX information model.
+- Offer sync and async clients to retrieve metadata
+  from an SDMX Registry or SDMX-REST service, and use them to
+  drive statistical business processes.
+- Offer SDMX-REST query builders and a service client to execute
+  queries against SDMX-REST services.
+- Offer functions to handle SDMX URNs.
```

Comment:
Maybe you could add a feature like support for reading and writing data and structure messages for formats X, Y and Z?

Reply:
Yes, will do